src/frontend/src/eth/components/send/SendForm.svelte

```diff
@@ -61,9 +62,9 @@
 
 		<ButtonGroup slot="toolbar">
 			<slot name="cancel" />
-			<button class="primary block flex-1" type="submit" disabled={invalid}>
+			<ButtonPrimary disabled={invalid}>
```

Comment:
What about calling it `Button` and having it's ui behavior, primary or secondary, as property?

Reply:
Yes!!!! I was planning of having ButtonSecondary, but that is way better